internal/disk/lvm.go

```diff
@@ -128,3 +132,12 @@ func (lv *LVMLogicalVolume) EnsureSize(s uint64) bool {
 	}
 	return false
 }
+
+// lvname returns a name for a logical volume based on the mountpoint.
+func lvname(path string) string {
+	if path == "/" {
+		return "rootlv"
+	}
+
+	return strings.ReplaceAll(path, "/", "") + "lv"
```

Comment:
Does this need to be unique? The mapping is not injective, so if uniqueness is required this won't work.

Reply:
Yeah, indeed. Good point.